Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM java:openjdk-8-alpine
+
+COPY ./target/cqlTranslationServer-1.0-SNAPSHOT-jar-with-dependencies.jar /app/
```

Comment:
It's slower to build that way and also makes production images larger, but should be fine.